README.md

```diff
@@ -91,11 +91,17 @@ var regResult = await f2l.attestationResult(clientAttestationResponse, attestati
 ``` js
 var authnOptions = await f2l.assertionOptions();
 
+// add allowCredentials if you want...
```

Comment:
This part is not clear to understand for a beginner

Reply:
I have added some explanation to the line.